change/@fluentui-web-components-94ca4f8e-0e57-42d1-80e3-d4e828c8d008.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
curious, how does this get created?

Reply:
This can be generated for PR's using `yarn change` at the root. Currently we only support "none" or "prerelease", but typically when not in alpha/beta it's got additional semver values.